### PR TITLE
fix screenshot references in Jupyter doc

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -171,6 +171,11 @@ image::execute.gif[100000,2000]
 The ActiveEon Jupyter Kernel adds a kernel backend to Jupyter. This kernel interfaces directly with the ProActive
 scheduler and constructs tasks and workflows to execute them on the fly.
 
+In seconds, you can now switch from executing your code on your local machine and on a remote server. Develop locally
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: Honestly, I tried to refactor and understand this paragraph but I couldn't. please refactor or fix.

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -591,6 +596,8 @@ The returned values of your final tasks will be automatically printed.
 Finally, to have the hand on more parameters and features, the user should use ActiveEon Studio portals.
 The main ones are the _Resource Manager_, the _Scheduling Portal_ and the _Workflow Automation_.
 
+The example below show how the user can directly see his submitted job being executed in the scheduling portal:
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: The example below shows how the user can directly display the running job in the scheduling portal:

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 3]
```diff
@@ -591,6 +594,8 @@ The returned values of your final tasks will be automatically printed.
 Finally, to have the hand on more parameters and features, the user should use ActiveEon Studio portals.
 The main ones are the _Resource Manager_, the _Scheduling Portal_ and the _Workflow Automation_.
 
+The example below shows how the user can directly monitor his submitted job's execution in the scheduling portal:
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: The example below shows how the user can directly monitor the execution of submitted jobs in the scheduling portal:

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 4]
```diff
@@ -171,6 +171,9 @@ image::execute.gif[100000,2000]
 The ActiveEon Jupyter Kernel adds a kernel backend to Jupyter. This kernel interfaces directly with the ProActive
 scheduler and constructs tasks and workflows to execute them on the fly.
 
+With this interface, a user can run his code locally and test it using a native python kernel, and by a simple switch to
+ProActive kernel, run it on remote public/private infrastructures without having to modify his code. See the example below:
+
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: Using this interface, users can run their code locally and test it using a native python kernel. For instance, a simple switch to the ProActive kernel allows running code on remote public or private infrastructures without having to modify the code. See the example below:

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 3 above)
Comment (mklkun) [reply src/docs/MLOS/MLOSUserGuide.adoc]: I prefer keeping the original sentence. It puts more emphasis on which execution and which job we are talking about.

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 4 above)
Comment (mklkun) [reply src/docs/MLOS/MLOSUserGuide.adoc]: Sorry but I don't agree here neither. Your suggestion changes the meaning of the original sentence. I will refactor it.
Thanks!

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 3 above)
Comment (mcompastie) [reply src/docs/MLOS/MLOSUserGuide.adoc]: `his submitted job's execution ` -> Too long.  Prop: `can directly monitor the submitted jobs during their execution phase in the ...`

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 4 above)
Comment (mcompastie) [reply src/docs/MLOS/MLOSUserGuide.adoc]: Your sentence is too long. Please consider rephrasing it into multiple ones.

**Pull request merged**